src/Elm/AST/Typed.elm

```diff
@@ -214,6 +215,9 @@ recurse fn locatedExpr =
                                     }
                                 )
                                 branches
+
+                    Shader shader ->
+                        Shader shader
```

Comment:
We could do `expr` instead of explicitly writing the value out, but that's just a small nitpick :)